saleor/graphql/core/types/converter.py

```diff
@@ -17,7 +17,10 @@
 
 
 def get_form_field_description(field):
-    return str(field.help_text) if field.help_text else None
+    if hasattr(field, "help_text"):
```

Comment:
IMO we could use `description` as the name of this field.

Reply:
`help_text` is standard attribute for Django models and thus it's somewhat familiar. I also saw it used frequently in saleor's filter definitions. I think changing that notation is out of scope here.